### DISCUSSION TITLE
Does 'Manage' > 'Settings' > 'Files' - 'File mask' supports multiple files?

### DISCUSSION THREAD

**Discussion by Maksym5732005** [WeblateOrg/weblate#11689] [Q&A] [answered]
Hi everybody!
I'm trying to translate my Android app and am stuck a bit.
There are two folders in my project where I have language files:
1) This works right now well:
androidApp/src/main/res/values-*/strings.xml
I have 3 languages there: en, de, fr
2) Need to add this one also:
androidApp/src/envDev/res/values-*/strings.xml
I have two languages there: no, it

In the project settings: Manage > Settings > Files
for the 'File mask' I have tried these solutions:
- androidApp/src/main/res/values-*/strings.xml, androidApp/src/envDev/res/values-*/strings.xml
- androidApp/src/main/res/values-*/strings.xml | androidApp/src/envDev/res/values-*/strings.xml
- androidApp/src/(main|envDev)/res/values-*/strings.xml
nothing works for me. I get alert: Check whether the file mask  ... matches files in your version control system repository.
How can I track multiple folders in my project?

**Comment by nijel** [comment, marked as answer]
No, only a single file mask can be used there.
